packages/security/src/components/PanelV2/PanelV2-story.js

```diff
@@ -64,6 +66,27 @@ const content = (
 );
 
 disableCenteredStories(storiesOf(patterns('PanelV2#legacy'), module))
+  .addDecorator((Story) => (
+    <>
+      <InlineNotification
```

Comment:
Another instances where it looks like the header in Storybook is covering the actual inline notification for the new component.